src/test/java/gov/healthit/chpl/aqa/stepDefinitions/OverviewTest.java

```diff
@@ -32,15 +42,14 @@ public void overviewPageLoadsSuccessfullyAndVerifyTitle() throws Throwable {
 
     @When("^User looks at ONC-ACB and ONC-ATL information table$")
     public void userLooksAtONCACBAndONCATLInformationTable() throws Throwable {
-        driver.get("https://chpl.ahrqstg.org/#/resources/overview");
-        System.out.println("ONC-ACB and ONC-ATL Information");
+        driver.get(url + "#/resources/overview");
```

Comment:
I think this step is unnecessary. Should probably be removed from the Feature file. In either case, the "act of looking" isn't the same as navigating to another page